exasol_integration_test_docker_environment/lib/test_environment/spawn_test_database.py

```diff
@@ -64,6 +64,7 @@ def __init__(self, *args, **kwargs):
 
         self.db_version = DbVersion.from_db_version_str(self.docker_db_image_version)
         self.docker_db_config_resource_name = f"docker_db_config/{self.db_version}"
+        self.ports = Ports.default_ports
```

Comment:
```suggestion
        self.docker_db_internal_ports = Ports.default_ports
```

Reply:
I would really prefer a shorter name. The class is already named `SpawnTestDockerDatabase`. Adding  `docker_db` to the name of the variable seems redundant to me.  [ Actually I'd party if one day we dare to drop the prefix "SpawnTest" :) ]

Regarding the `internal` I'd prefer to use a prefix or different name for the _forwarded_ ports.
Currently those are called `port_mappings`.
So from my perspective the naming is clear, short and precise:
* `ports`
* vs `port_mappings`

But please: feel free to challenge my proposal.

Reply:
1. I think, for the parts where we know the changes to name, you can do that in a separate PR. SpawnTest is probably the first candidate.
2. I think, internal_ports and port_forwards is fine. This probably also what we can use for the DatabaseInfo
3. Port_mappings is if I remember correctly, the mapping we give to the docker API and is completely separate from that 
4.

Reply:
I will rename `self.ports` to `self.internal_ports`.

Optional follow-up-changes
* Switch from separate luigi parameters to `JsonPickleParameter(Ports, significant=False)`

Reply:
See #356